Quantum-Control-Applications/ESR/T1.py

```diff
@@ -0,0 +1,172 @@
+from qm import SimulationConfig
+from qm.qua import *
+from qm import LoopbackInterface
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+import matplotlib.pyplot as plt
+from qm.simulate.credentials import create_credentials
+
+
+###################
+# The QUA program #
+###################
+
+pi_len = 320 // 4
+pi_half_len = 160 // 4
+
+wait_min = 1500 // 4
+wait_max = 20000 // 4
+dwait = 1000 // 4
+
+waits = np.arange(wait_min, wait_max + 0.1, dwait)
+
+cooldown_time = int(10e6 // 4)
+
+n_avg = 100
+
+pulse_delay_p = int(2000 // 4 - pi_half_len * 0.5 - pi_len * 0.5)
+readout_delay_p = int(2000 // 4 - pi_len * 0.5 - readout_len * 0.125 - 5)
```

Comment:
I would explain in comments why we have to wait these specific values, especially why 2000ns? and why readout_len*0.125-5?